internal/services/logic/logic_app_integration_account_partner_resource.go

```diff
@@ -0,0 +1,227 @@
+package logic
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/logic/mgmt/2019-05-01/logic"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/logic/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/logic/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+func resourceLogicAppIntegrationAccountPartner() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceLogicAppIntegrationAccountPartnerCreateUpdate,
+		Read:   resourceLogicAppIntegrationAccountPartnerRead,
+		Update: resourceLogicAppIntegrationAccountPartnerCreateUpdate,
+		Delete: resourceLogicAppIntegrationAccountPartnerDelete,
+
+		Timeouts: &pluginsdk.ResourceTimeout{
+			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
+			Update: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Delete: pluginsdk.DefaultTimeout(30 * time.Minute),
+		},
+
+		Importer: pluginsdk.ImporterValidatingResourceId(func(id string) error {
+			_, err := parse.IntegrationAccountPartnerID(id)
+			return err
+		}),
+
+		Schema: map[string]*pluginsdk.Schema{
+			"name": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.IntegrationAccountPartnerName(),
+			},
+
+			"resource_group_name": azure.SchemaResourceGroupName(),
+
+			"integration_account_name": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.IntegrationAccountName(),
+			},
+
+			"business_identity": {
```

Comment:
this should be plural?
```suggestion
			"business_identities": {
```

Reply:
I assume here should be singular. I assume the field name should be plural when the situation is below this. If I am wrong, please correct me.

```
"match_values": {
	Type:     pluginsdk.TypeSet,
	Required: true,
	Elem: &pluginsdk.Schema{
		Type: pluginsdk.TypeString,
	},
},
```

Reply:
👍